### PR TITLE
vscode-insiders: Change to a static url and hashes

### PR DESCRIPTION
**Pull request by alexchapar**
## Description

VSCode-Insiders seems to use a "canary-build" version and new "file" replace the current version of file (for example 1.57.0-insider). Because of that, since the URL and version didn't changed the hashes wasn't updated by the "excavator Actions".

This "tweak" change finds the exact location of a version and will use it until the next version changes (minor, major or patch)

Also, we changed the registry files location to use this repository instead of an external

## Related Tickets & Documents
Not related

### Check Hashes?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Check Url?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Check the last version?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added a description in the manifest?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Beautified the json manifest format?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added some notes in the manifest?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added/Needed any dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## Added/Needed any pre-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## Added/Needed any post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?

:disappointed:

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
[Your changes do not pass checks.](https://github.com/Ash258/Scoop-GithubActions/wiki/Pull-Request-Checks)
### vscode-insiders

- [x] Description
- [x] License
- [ ] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### vscode-insiders

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87-CG — APPROVED** [review]

**Pull request merged**
